pkg/scheduler/ugm/queue_tracker.go

```diff
@@ -33,71 +31,163 @@ type QueueTracker struct {
 	queueName           string
 	resourceUsage       *resources.Resource
 	runningApplications map[string]bool
-	maxResourceUsage    *resources.Resource
+	maxResources        *resources.Resource
 	maxRunningApps      uint64
 	childQueueTrackers  map[string]*QueueTracker
 }
 
 func newRootQueueTracker() *QueueTracker {
-	return newQueueTracker(configs.RootQueue)
+	qt := newQueueTracker(configs.RootQueue)
+	return qt
 }
 
 func newQueueTracker(queueName string) *QueueTracker {
```

Comment:
new signature: newQueueTracker(queuePath, queueName string) *QueueTracker
add the queuePath filed with the full path to the object:
```
qt.queuePath := queuePath + "." + queueName
qt.queueName := queueName
```
the call for the root queue gets an empty path:
```
rootQueue := newQueueTracker("", configs.RootQueue)
```
when we create a new child we use the current path and the childname:
```
qt.childQueueTracker[child] = newQueueTracker(qt.path, childName)
```
Looking up the queue in the wildcard list:
```
wildcardQuota := getUserWildCardLimitsConfig(qt.path)
```